### DISCUSSION TITLE
New config option in 6.5.0

### DISCUSSION THREAD

**Discussion by gboudreau** [cross-seed/cross-seed#845] [General]
Added in 6.5.0: https://www.cross-seed.org/docs/basics/options#seasonfromepisodes
This option is enabled by default, and cause cross-seed to not be able to start, if you have no `linkDir` defined.
Also, the documentation says setting it to undefined or null will disable the feature - that doesn't seem to be the case for undefined. It stills complains on boot about invalid config. Setting it to null does work (disables it).

`seasonFromEpisodes: undefined` (or not present in config) result in:
```
Configuration file already exists.
2024-12-21 03:23:24 info: cross-seed v6.5.0
2024-12-21 03:23:25 info: Validating your configuration...
2024-12-21 03:23:25 warn: outputDir should only contain .torrent files, cross-seed will populate and manage (https://www.cross-seed.org/docs/basics/options#outputdir)
2024-12-21 03:23:25 error: Configuration:
				You need to use the 'inject' action for seasonFromEpisodes.
				(https://www.cross-seed.org/docs/basics/options)

2024-12-21 03:23:25 error: Configuration:
				When using action 'inject', you need to set a linkDir (and have your data accessible) for risky/partial matching and seasonFromEpisodes.
				(https://www.cross-seed.org/docs/basics/options)

2024-12-21 03:23:25 error: Your configuration is invalid, please see the errors above for details.
```

**Comment by zakkarry** [comment]
undefined is not the same as null, the docs are outdated, i'm about to push an update.

the default is 100% completion of combined episodes will be cross-seeded

you cannot combine episodes into seasons without linking them to a new path - thus requiring a linkDir.

use null.

**Comment by zakkarry** [comment]
I've updated the documentation, thank you for pointing this out.

We will be adding a CLI argument for making it null as well shortly.
